jekyll/_cci2/notifications.md

```diff
@@ -149,4 +149,8 @@ workflows:
       - build
 ```
 
+## Notification with RGB hardware
 
+[Chroma Feedback](https://github.com/redaxmedia/chroma-feedback) is a command line tool in Python to turn your RGB powered hardware into an build indicator. The idea of such extreme visibility is to encourage developers to instantly repair their builds.
+
+![Terminal Session](https://cdn.rawgit.com/redaxmedia/media/master/chroma-feedback/terminal-session.svg)
```

Comment:
Can you remove this please? We try to reduce gif usage in our docs. I think having a link to the project should suffice?

Reply:
No problem...